### PR TITLE
[CSFS-95] Loops and Numbers

### PR DESCRIPTION
**Pull request by ToluEnny24**
I have created a PHP program for numbers and loops

### CODE REVIEW

File: submissions/fabunmiTolulope/php/Number_loops.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+// This program will display every number between 100 and 150 using while loop
+$x = 100;
+while ($x <= 150) {
+	echo "Number: $x <br>";
+	$x++;
+}
+
+// This program will display even numbers between 0 and 50 using for loop
+for($y = 0; $y < 50; $y++){
+    if($y > 0 && $y % 2 === 0){
```
Comment (adaugochi) [inline submissions/fabunmiTolulope/php/Number_loops.php:12]: There is no need to do this check (`$y > 0`) if your start your count from 2

File: submissions/fabunmiTolulope/php/Number_loops.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+// This program will display every number between 100 and 150 using while loop
+$x = 100;
+while ($x <= 150) {
```
Comment (adaugochi) [inline submissions/fabunmiTolulope/php/Number_loops.php:5]: The question says between `100 and 150`

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@ToluEnny24 There are comments that needs to be worked on. Are you having issues figuring them out?

File: submissions/fabunmiTolulope/php/Number_loops.php (hunk 2 above)
Comment (ToluEnny24) [reply submissions/fabunmiTolulope/php/Number_loops.php:5]: I will effect the changes as soon as I can... @adaugochi

**Comment by ToluEnny24** [comment]
@ToluAkin  I had to take a break, my laptop crashed again and currently under repair... I will continue as soon as I can. Thanks

File: submissions/fabunmiTolulope/php/Number_loops.php (hunk 2 above)
Comment (ToluAkin) [reply submissions/fabunmiTolulope/php/Number_loops.php:5]: Between includes 100 and 150

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/fabunmiTolulope/php/Number_loops.php (hunk 2 above)
Comment (ToluEnny24) [reply submissions/fabunmiTolulope/php/Number_loops.php]: @ToluAkin I have changed it from <= to <. Thanks

**Review by ToluAkin — APPROVED** [review]

**Pull request merged**